### DISCUSSION TITLE
Whether it is possible to build in awes-chatGPt-prompts

### DISCUSSION THREAD

**Discussion by a13782425** [danny-avila/LibreChat#2758] [General]
build in awes-chatGPt-prompts，Use / for input.
like
<img width="953" alt="208471439-877c2bcf-93ec-4ad9-9cb0-7e4ed7b1756a" src="https://github.com/danny-avila/LibreChat/assets/11312875/df9bd1dc-659c-40d7-86f8-f5eb479d19cf">

**Comment by danny-avila** [comment]
prompts are on the roadmap and they will be able to be used like this for sure
